pkg/services/pluginsintegration/plugininstaller/service.go

```diff
@@ -97,6 +122,9 @@ func (s *Service) installPlugins(ctx context.Context) error {
 			continue
 		}
 		s.log.Info("Plugin successfully installed", "pluginId", installPlugin.ID, "version", installPlugin.Version)
+		elapsed := time.Since(start)
```

Comment:
Would include the elapsed time in the log. Almost thinking logs would be enough for this use case (on-prem) 🤷

Reply:
yeah, that's what we discussed in [here](https://github.com/grafana/grafana/pull/91790#discussion_r1714900422) but this could be helpful in a future escalation or if we decide to start using this in Cloud.